vector/array.go

```diff
@@ -38,3 +38,33 @@ func (a *Array) Serialize(b *zcode.Builder, slot uint32) {
 	}
 	b.EndContainer()
 }
+
+func OffsetsOf(val Any, slot uint32) (uint32, uint32, bool) {
+	// Dict / Const do not appear for Array, Sets, Maps?
```

Comment:
@mccanne am I right here in that Dict and Consts are not used for Arrays? Also the function name is kind of weird if you have any better ideas.